src/crate/client/tests.py

```diff
@@ -108,15 +111,33 @@ def setUpMocked(test):
 crate_port = 44209
 crate_transport_port = 44309
 local = '127.0.0.1'
-crate_layer = CrateLayer('crate',
-                         crate_home=crate_path(),
-                         port=crate_port,
-                         host=local,
-                         transport_port=crate_transport_port,
-                         settings=settings)
-
 crate_host = "{host}:{port}".format(host=local, port=crate_port)
 crate_uri = "http://%s" % crate_host
+crate_layer = None
+
+
+def ensure_cratedb_layer():
+    """
+    In order to skip individual tests by manually disabling them within
+    `def test_suite()`, it is crucial make the test layer not run on each
+    and every occasion. So, things like this will be possible::
+
+        ./bin/test -vvvv --ignore_dir=testing
+
+    TODO: Through a subsequent patch, the possibility to individually
```

Comment:
We could use the logic from here https://github.com/crate/crate/commit/414cd8339b216478a7a06b10e4c6353dba7cfc74

Reply:
Oh this is great. That is exactly what I had in mind here. With #386, the need for having this immediately got a lower priority for me, but I will come back to this as soon as we will need it.

Reply:
I added your suggestion on behalf of 3335961f29. Thanks again!